gm4_ink_spitting_squid/data/gm4/advancements/glow_ink_spitting_squid.json

```diff
@@ -0,0 +1,29 @@
+{
+  "display": {
+    "icon": {
+      "item": "minecraft:glow_ink_sac",
+      "nbt": "{CustomModelData:1}"
+    },
+    "title": {
+      "translate": "%1$s%3427655$s",
+      "with": [
+        "The Light, It Burns!",
+        {"translate": "advancement.gm4.glow_ink_spitting_squid.title"}
+      ]
+    },
+    "description": {
+      "translate": "%1$s%3427655$s",
+      "with": [
+        "Glow due to an underwater glow squid",
+        {"translate": "advancement.gm4.glow_ink_spitting_squid.description"}
+      ],
+      "color": "gray"
+    }
+  },
+  "parent": "gm4:root",
```

Comment:
I would recommend making this a child of the normal squid advancement, and perhaps making it a "goal" type frame as glow squids are slightly more rare